src/components/modal/protectKeys/index.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import { OLD_ORD_PK_KEY, OLD_PAY_PK_KEY } from "@/constants";
+import {
+	ProtectKeysStep,
+	bsvWasmReady,
+	protectKeysStep,
+} from "@/signals/wallet";
+import { setKeys } from "@/signals/wallet/client";
+import { useSignals } from "@preact/signals-react/runtime";
+import { useEffect } from "react";
+import { DoneStep } from "./steps/DoneStep";
+import { EnterPassphraseStep } from "./steps/EnterPassphraseStep";
+import { InfoStep } from "./steps/InfoStep";
+
+const ProtectKeysModal = ({
+	open,
+	onClose,
+}: {
+	open: boolean;
+	onClose: () => void;
+}) => {
+	useSignals();
+
+	function handleClose() {
+		protectKeysStep.value = ProtectKeysStep.Info;
+		onClose();
+	}
+
+	useEffect(() => {
+		const payPk = localStorage.getItem(OLD_PAY_PK_KEY);
+		const ordPk = localStorage.getItem(OLD_ORD_PK_KEY);
+
+		if (!payPk || !ordPk) {
+			return;
+		}
+
+		setKeys({ payPk, ordPk });
```

Comment:
needs to be 
setKeys({ payPk: JSON.parse(payPk), ordPk: JSON.parse(ordPk) });
to parse the existing keys in localStorage

Reply:
@rohenaz thank you, pushed a fix 👍